src/fiatExchanges/ExternalExchanges.tsx

```diff
@@ -44,8 +38,6 @@ type Props = NativeStackScreenProps<StackParamList, Screens.ExternalExchanges>
 
 function ExternalExchanges({ route }: Props) {
```

Comment:
since this component no longer works for cash ins, it might be helpful to rename it to make that more clear for potential consumers